documentation/WiFi_Direct_Support.md

```diff
@@ -0,0 +1,20 @@
+## Wi-Fi Direct Support
+
+[Wi-Fi Direct](http://www.wi-fi.org/discover-wi-fi/wi-fi-direct) allows devices to connect directly rather than through an Access Point. This way, a device can connect and received data through a peer-to-peer connection rather than over the internet. With Wi-Fi Direct, the Physical Web will allow beacons to broadcast and serve data to users. The advantage this provides is allowing devices without internet connectivity to access data. For more informatio
+
+Below are instructions on how to setup any capable device to broadcast Physical Web data using Wi-Fi Direct.
+
+To "broadcast", a device must be in a Wi-Fi Direct discoverable state.
+In addition, the device name must conform to the following naming convention:
+
+PW-title-port
+
+"title" should be replaced by a fitting title for the content you wish to provide. The port should be where you will be serving HTTP data, whether that be HTML or other types.
```

Comment:
`<title>` is the title of the content.  `<port>` is the port the HTTP service is running at.